### PR TITLE
Add OCR validation warnings column in DB

### PR DESCRIPTION
**Pull request by lgonczar**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/BPS-759

### Change description ###

Add OCR validation warnings column in DB

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/db/migration/V044__Add_validation_warnings_column.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ALTER TABLE scannable_items
+ADD COLUMN ocrValidationWarnings VARCHAR(1000)[] NULL;
```
Comment (nitinprabhu) [inline src/main/resources/db/migration/V044__Add_validation_warnings_column.sql]: not sure if varchar(1000) is sufficient or should be keep it bit more ?

Comment (lgonczar) [reply src/main/resources/db/migration/V044__Add_validation_warnings_column.sql]: How about 5k?

Comment (rkondratowicz) [reply src/main/resources/db/migration/V044__Add_validation_warnings_column.sql]: Seems more than enough for a single warning message to me

Comment (rkondratowicz) [reply src/main/resources/db/migration/V044__Add_validation_warnings_column.sql]: Sample 1000 char string:
>mi quis hendrerit dolor magna eget est lorem ipsum dolor sit amet consectetur adipiscing elit pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas integer eget aliquet nibh praesent tristique magna sit amet purus gravida quis blandit turpis cursus in hac habitasse platea dictumst quisque sagittis purus sit amet volutpat consequat mauris nunc congue nisi vitae suscipit tellus mauris a diam maecenas sed enim ut sem viverra aliquet eget sit amet tellus cras adipiscing enim eu turpis egestas pretium aenean pharetra magna ac placerat vestibulum lectus mauris ultrices eros in cursus turpis massa tincidunt dui ut mi quis hendrerit dolor magna eget est lorem ipsum dolor sit amet consectetur adipiscing elit pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas integer eget aliquet nibh praesent tristique magna sit amet purus gravida quis blandit turpis cursus in hac habitasse platea dictumst quisque sagittis purusus

Comment (lgonczar) [reply src/main/resources/db/migration/V044__Add_validation_warnings_column.sql]: and that's per warning

**Comment by lgonczar** [comment]
Closing, as the change will not be applied within the same PR

**Pull request closed without merging**